.github/workflows/tests.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [20, 18]
+        node: [22, 18]
```

Comment:
let's use the two latest LTS versions (24,22)

Reply:
i'll bump package.json then